ts-sdk/whirlpool/src/token.ts

```diff
@@ -279,3 +279,14 @@ export function getCurrentTransferFee(
     maxFee: transferFee.maximumFee,
   };
 }
+
+/**
+ * Orders two mints by canonical byte order.
+ * @param {Address} mint1
+ * @param {Address} mint2
+ * @returns {[Address, Address]} [mint1, mint2] if mint1 should come first, [mint2, mint1] otherwise
+ */
+export function orderMints(mint1: Address, mint2: Address): [Address, Address] {
+  const encoder = getAddressEncoder();
+  return encoder.encode(mint1) < encoder.encode(mint2) ? [mint1, mint2] : [mint2, mint1];
```

Comment:
`<` will try to convert operands to string ("[20, 3, 4]", "[1, 2, 3]").

```
   const array1 = new Uint8Array([1, 2, 3]);
   const array2 = new Uint8Array([2, 3, 4]);
   console.log("array1 < array2", array1 < array2);

   const array3 = new Uint8Array([100, 2, 3]);
   const array4 = new Uint8Array([2, 3, 4]);
   console.log("array3 < array4", array3 < array4);

   const array5 = new Uint8Array([3, 4, 5]);
   const array6 = new Uint8Array([20, 3, 4]);
   console.log("array5 < array6", array5 < array6);
```

Reply:
Interesting. So buffer.compare works differently than '<'. I'll copy the legacy code cause we know that works

Reply:
memo: `Buffer.compare` can be used for both `Buffer` and `Uint8Array`
https://nodejs.org/api/buffer.html#static-method-buffercomparebuf1-buf2